<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRcodeDialog</class>
 <widget class="QDialog" name="QRcodeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>二维码生成小工具</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>11</y>
     <width>901</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="MainLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_left" native="true">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>431</width>
         <height>511</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="LeftLayout">
        <item>
         <widget class="QPlainTextEdit" name="edit_content"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btn_generate">
            <property name="text">
             <string>生成</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_clear">
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_right" native="true">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>401</width>
         <height>471</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="RightLayout" stretch="1,0">
        <item>
         <widget class="QLabel" name="label_qrcode">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btn_open">
            <property name="text">
             <string>打开所在位置</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_save_as">
            <property name="text">
             <string>另存为</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_copy">
            <property name="text">
             <string>复制图片</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
